lib/apollo-federation/tracing.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module ApolloFederation
+  module Tracing
+    KEY = :ftv1
+
+    def self.use(schema)
+      schema.tracer ApolloFederation::Tracing::Tracer
+    end
+
+    def self.should_add_traces(headers)
+      headers && headers['apollo-federation-include-trace'] == KEY.to_s
```

Comment:
Why is it expected for this header to be set to the key instead of just a boolean value?

Reply:
https://www.apollographql.com/docs/apollo-server/federation/metrics/#how-tracing-data-is-exposed-from-a-federated-service